WordPress/src/main/java/org/wordpress/android/ui/posts/PostUploadAction.kt

```diff
@@ -45,15 +46,21 @@ sealed class PostUploadAction {
     class CancelPostAndMediaUpload(val post: PostModel) : PostUploadAction()
 }
 
-fun handleUploadAction(action: PostUploadAction, activity: Activity, snackbarAttachView: View) {
+fun handleUploadAction(
+    action: PostUploadAction,
```

Comment:
This argument and class name confused me for a few minutes. I thought it was the `UploadAction`. 🤪

Reply:
:D yes, you are right.. That's quite confusing.